pkg/streamingpromql/operators/binary_operation_test.go

```diff
@@ -1130,7 +1135,10 @@ func TestBinaryOperationSeriesBuffer(t *testing.T) {
 	}
 
 	seriesUsed := []bool{true, false, true, true, true}
-	buffer := newBinaryOperationSeriesBuffer(inner, seriesUsed, limiting.NewMemoryConsumptionTracker(0, nil))
+	memoryConsumptionTracker := limiting.NewMemoryConsumptionTracker(0, nil)
+	// We have 6 FPoints from the inner series
+	memoryConsumptionTracker.IncreaseMemoryConsumption(types.FPointSize * 6)
```

Comment:
Why is this needed?

Reply:
See other comment. Will resolve this one and move conversation there.